### ISSUE TITLE
[UI] Offline mode status message overlaps with the latest item in the list

### ISSUE THREAD

**Issue by jesmrec** [owncloud/ios-app#209]
Offline mode status message hides belowest item in the file list 

![screen shot 2019-01-14 at 13 32 50](https://user-images.githubusercontent.com/14894746/51113033-02a65580-1801-11e9-89c4-c75daac97917.png)

List can be pulled up to check the info (as workaround), but other design solutions can be discussed

**Comment by michaelstingl** [comment]
@jesmrec can't be scrolled up?

**Comment by jesmrec** [comment]
yes it is. this is the "workaround" i mentioned. if the all the team considers it enough, then we can close.

**Comment by michaelstingl** [comment]
So, needs only a few pixel padding at the bottom? Shouldn't be that hard?

**Comment by mneuwert** [comment]
Also `CollapsibleProgressBar` sometimes is temporarily shown while transitioning from the list view into PDF or image gallery view. It looks quite ugly since it is just overlayed on top the image or PDF document. There must be a way to auto-collapse it immediately, when the file list view is left.
